ansible/templates/crd.yml.j2

```diff
@@ -151,6 +151,8 @@ spec:
                         type: string
                 control_plane_ee_image:
                   description: Registry path to the Execution Environment container image to use on control plane pods
```

Comment:
@Zokormazo so I think when I did the rebase I messed this up. all these need to have a type, right?